loops_lab.js

```diff
@@ -0,0 +1,173 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
No major feedback here. I think you understood the concept and implementation of loops clearly. Keep up the good work!